### DISCUSSION TITLE
Reset Netbox?

### DISCUSSION THREAD

**Discussion by fpausp** [netbox-community/netbox#7251] [General]
Hi Folks,

I would like to clone my netbox VM (ubuntu 20.04) and reset it later on to get a fresh installed server.
Can someone tell my the commands for the CLI please?

**Comment by candlerb** [comment]
After cloning, you can reset the instance like this:

1. Drop and recreate the database
    * Instructions [here](https://netbox.readthedocs.io/en/stable/installation/1-postgresql/#database-creation)
    * Do "DROP DATABASE netbox;" first
2. Re-run the upgrade.sh script, as per [here](https://netbox.readthedocs.io/en/stable/installation/3-netbox/#run-the-upgrade-script)
3. Delete the contents of your media directory, if any

You might also find it helpful to review the contents of [replicating netbox](https://netbox.readthedocs.io/en/stable/administration/replicating-netbox/) which is related: given a freshly installed Netbox instance, how to wipe its database and populate it with data backed up from a live server

**Reply by fpausp** [reply]
Hello and Thank you! - This worked for me...
